### PR TITLE
VITA: proper handling of boot params

### PR DESCRIPTION
**Pull request by cy33hc**
Properly handle the boot params for buffer overflow

@twinaphex @rsn8887 @crystalct Please review the changes for handling boot params

### CODE REVIEW

**Comment by twinaphex** [comment]
Can you use C comments here instead of C++ comments? Thanks.

**Comment by cy33hc** [comment]
> Can you use C comments here instead of C++ comments? Thanks.

Updated

**Comment by twinaphex** [comment]
@rsn8887 Can you review this please?

File: frontend/drivers/platform_psp.c [hunk 1]
```diff
@@ -351,21 +350,47 @@ static void frontend_psp_exec(const char *path, bool should_load_game)
...
+            RARCH_LOG("Boot params are too long. Continue normal boot.");
+         }
       }
-      RARCH_LOG("Required boot params missing. Continue nornal boot.");
+      else
+      {
+         RARCH_LOG("Required boot params missing. Continue nornal boot.");
+      }
+      
+      int ret =  sceAppMgrLoadExec(path, args == 0 ? NULL : (char * const*)((const char*[]){argp, 0}), NULL);
```
Comment (frangarcj) [inline frontend/drivers/platform_psp.c:385]: could you refactor code to just use this sceAppMgrLoadExec call?

I see too much duplicated code

**Review by frangarcj — CHANGES_REQUESTED** [review]

File: frontend/drivers/platform_psp.c (hunk 1 above)
Comment (cy33hc) [reply frontend/drivers/platform_psp.c]: @frangarcj refactored to remove duplicate code

File: frontend/drivers/platform_psp.c [hunk 2]
```diff
@@ -349,23 +348,49 @@ static void frontend_psp_exec(const char *path, bool should_load_game)
...
+   else
 #endif
-   ret =  sceAppMgrLoadExec(path, args == 0 ? NULL : (char * const*)((const char*[]){argp, 0}), NULL);
-   RARCH_LOG("Attempt to load executable: [%d].\n", ret);
-exit:
-   return;
+   {
+      int ret =  sceAppMgrLoadExec(path, args == 0 ? NULL : (char * const*)((const char*[]){argp, 0}), NULL);
+      RARCH_LOG("Attempt to load executable: [%d].\n", ret);
+   }
```
Comment (frangarcj) [inline frontend/drivers/platform_psp.c:393]: Last thing, use just this load exec with core name and remove previous one

**Review by cy33hc — COMMENTED** [review]
@frangarcj @twinaphex If you're ok with current change. Can this be approved?

**Pull request merged**